contracts/eoslib/eosc.dox

```diff
@@ -6,7 +6,12 @@
   @section intro Introduction to EOSC
   
   `eosc` is a command line tool that interfaces with the REST api exposed by @ref eosd. In order to use `eosc` you will need to
-  have a local copy of `eosd` running and configured to load the 'eos::chain_api_plugin'.  
+  have a local copy of `eosd` running and configured to load the 'eos::chain_api_plugin'. Currently, `eosc` does not
```

Comment:
This documentation misrepresents what is going on.  It has nothing to do with eosc support for signature checking and everything to do with making it easy for developers to test functions without keys.